modules/ROOT/pages/setup.adoc

```diff
@@ -53,42 +53,72 @@ To enable the Anypoint Monitoring agent:
-. Next to application, click *Enable & Apply* to enable visualization and monitoring for the application.
+. In *List of resources in <Environment Name>*, search for an application to visualize.
+. Next to the application, click *Enable & Apply* to enable visualization and monitoring for the application.
 
 For more information about Anypoint Monitoring, see xref:monitoring::enable-apps-deployed-to-cloud.adoc[Enable Monitoring Using the UI].
 
-== Set up for Standalone Mule 
+== Setup for Standalone Mule and Runtime Fabric
 
-=== Verify Supported Standalone Mule Runtime Engine (Mule) Versions
+=== Supported Standalone Mule and Runtime Fabric Mule Versions
 
-Anypoint Visualizer supports the following Mule versions for standalone Mule deployments:
+Anypoint Visualizer supports the following Mule versions for standalone Mule and Runtime Fabric deployments:
 
-* Mule Runtime, version 3.8.7
-* Mule Runtime, version 3.9.1+
-* Mule Runtime, version 4.1.1+
+[%header,cols="2*a"]
+|===
+|Deployment Type| Supported Mule versions
+| Mule standalone 
+a| * Mule 3.8.7
+* Mule 3.9.1+
+* Mule 4.1.1+ +
+
+| Runtime Fabric
+a| * Mule 3.8.7
+* Mule 3.9.1+
+* Mule 4.1.2+ +
+|===
+
+[NOTE]
+By default, Visualizer is disabled for Mule 4. See <<enable_header_injection,enable the header injection>> for more information. 
 
 === Standalone Mule Setup
 
 To support Visualizer for standalone Mule deployments, xref:monitoring::am-installing.adoc[install the Anypoint Monitoring agent]. 
 
+=== Runtime Fabric Setup
+
+Follow the instructions for xref:runtime-fabric::install-create-rtf-arm.adoc[setting up Runtime Fabric]. After you install Runtime Fabric, Visualizer can display applications that are running and deployed to Runtime Fabric.
+
+Visualizer relies on a header injection to display connections between services correctly. The header injection is disabled by default for Mule 4. The correct connections will not be displayed for your Mule 4 Runtime Fabric applications unless you <<enable_header_injection,enable the header injection>>. 
+
+Known Limitations:
+
+* If a Mule 3 CloudHub application with the old Visualizer agent (version earlier than April 5, 2019) is calling a Runtime Fabric ingress endpoint, there will be extra edge from the CloudHub app to a node with the IP address for the controller. Please upgrade to the latest release of the Mule runtime engine.
+* If a Mule 4 CloudHub application is calling a Runtime Fabric ingress endpoint, there will be and extra edge from the CloudHub app to a node with the IP address for the controller. You can use your filters within Visualizer to hide this node.
+* If a Runtime Fabric Mule app is calling a Mule 3 CloudHub Mule app with the old Visualizer agent (version earlier than April 5, 2019) using its DNS, the correct connection will be drawn, but there will be an additional edge from the *External Traffic* node to the CloudHub application.
```

Comment:
global: the correct connection is drawn (helps with the contrast for "there will be an additional edge" which is probably best in passive construction).